meshery

```diff
@@ -72,35 +83,42 @@ case $1 in
     --help) print_help ;;
 
     start)
-        if [ ! -f ./docker-compose.yaml ]; then
-            echo "Downloading Meshery's docker-compose.yml file..."
-            `$DOWNLOADCMD https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml`
-        fi
-        if [ ! -f ./docker-compose.yaml ]; then
+        if [ ! -f $dockerComposeFile ]; then
             echo "Unable to download Meshery's docker-compose.yaml file. Please try again later."
         else
             echo "Starting Meshery now..."
-            docker-compose up -d && \
+            docker-compose -f $dockerComposeFile up -d && \
             echo "Opening Meshery in your broswer. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery."
             [[ -z $BROWSERCMD ]] >/dev/null 2>&1 || `$BROWSERCMD http://localhost:9081`
         fi
     ;;
 
     stop)
-        echo "Stopping Meshery. . ."
-        docker-compose stop
-        (docker-compose rm -f) || true
-        (docker volume prune -f) || true
-        echo "Meshery is now stopped"
+        if [ ! -f $dockerComposeFile ]; then
+            echo "Unable to download Meshery's docker-compose.yaml file. Please try again later."
+        else
+            echo "Stopping Meshery. . ."
+            docker-compose -f $dockerComposeFile stop
+            (docker-compose -f $dockerComposeFile rm -f) || true
+            (docker volume prune -f) || true
+            echo "Meshery is now stopped"
+        fi
     ;;
     logs)
-        [[ -z $DOCKER_COMPOSE_VERSION ]] >/dev/null 2>&1 || echo>$2 "Missing 'docker-compose'. Please install and try again." ;
-        docker-compose logs -f
+        if [ ! -f $dockerComposeFile ]; then
+            echo "Unable to download Meshery's docker-compose.yaml file. Please try again later."
+        else
+            docker-compose -f $dockerComposeFile logs -f
+        fi
     ;;
     update)
-        echo "Updating Meshery..."
-        docker-compose pull
-        echo "Meshery is now up-to-date."
+        if [ ! -f $dockerComposeFile ]; then
+            echo "Unable to download Meshery's docker-compose.yaml file. Please try again later."
```

Comment:
Potentially, confusing message given that, yes, the user is downloading docker-compose.yaml, but doesn't know that we're renaming it to meshery.yaml necessarily. Maybe best to rename docker-compose.yaml in our repo to meshery.yaml.

Reply:
We do need folks who can cloned the repo to be able to run `docker-compose up -d`. If we change the file name then they will have to add a `-f <file name>`. For now I am leaving it as it is but have updated the error message to just say "config".